internal/pkg/application/transform/transformers.go

```diff
@@ -166,7 +166,12 @@ func IndoorEnvironmentObserved(ctx context.Context, msg core.MessageAccepted, cb
 		properties = append(properties, decorators.Number("illuminance", illuminance))
 	}
 
-	if !tempOk && !humidityOk && !illuminanceOk {
+	peopleCount, peopleCountOk := core.Get[float64](msg, PeopleCountURN, SensorValue)
```

Comment:
SensorValue här (5700) representerar ID 5700 i temp, humidity och illuminance. För People Counter så är ID 1 och heter "Actual number of persons". Du måste alltså skapa en egen konstant här (const ActualNumberOfPersons int = 1) och använda den i stället för SensorValue.